### ISSUE TITLE
Installer dont work

### ISSUE THREAD

**Issue by optimumpos** [opencart/opencart#14014]
**What version of OpenCart are you reporting this for?**
Version 4.0.2.3

**Describe the bug**
Installer page dont work, when we go to Extension >> Installer
We get the error.html page

**To Reproduce**
1. Go to Extention
2. Click on Installer

**Expected behavior**

**Screenshots / Screen recordings**
https://www.loom.com/share/c5ed0047fef44eeda14322b6661ed434?sid=1f6b45e1-e722-4e2d-89c9-bdae2cd3a9ae

**Server / Test environment (please complete the following information):**
 - Local development? Deployed to a web server?
 Deployed to airsoftstore.nl
 - Operating system
 Direct Admin
linux_amd64
OS: rhel7_amd64
 - PHP version
 PHP 8.1.29
 - Apache version
2.4.59
 - Browser(s) tested with [e.g. chrome, safari - if applicable]
 Chrome

**Additional context**

**Comment by danielkerr** [comment]
i think ur server is blocking ur php to grab the page

**Comment by optimumpos** [comment]
i get 

Undefined constant ZipArchive::RDONLY: in /home/admin/domains/airsoftarmy.nl/public_html/taner/controller/marketplace/installer.php on line 112

**Comment by arnaudjaegers** [comment]
Hello @optimumpos, you may have a PHP version which has not the libzip ≥ 1.0.0.

> ZipArchive::RDONLY (int)
> Open archive in read only mode. Available as of PHP 7.4.3 and PECL zip 1.17.1, respectively, if built against libzip ≥ 1.0.0.
> https://www.php.net/manual/en/zip.constants.php#ziparchive.constants.rdonly

You can easily obtain the libzib version with that tiny code:
```php
<?php
echo "\nCurrent PHP version: " . phpversion();
echo "\nZipArchive::LIBZIP_VERSION= ".ZipArchive::LIBZIP_VERSION;
echo "\n";
var_dump(ZipArchive::RDONLY);
```

Solution on our side was to update server OS.
